packages/example/src/pages/guides/navigation/sidebar.mdx

```diff
@@ -34,6 +34,25 @@ Some important things to note here:
 - You can make a `Page/index.mdx` file if you’d prefer to have assets in a folder.
   The path would still just look like `/Page`
 
+## Add divider
+
+You can add divider(s) between the items of your nav item list by adding `hasDivider: true` in `src/data/nav-items.yaml`.
```

Comment:
```suggestion
You can add divider(s) between the items of your nav item list by adding `hasDivider: true` in `src/data/nav-items.yaml`. Note the divider is only for top-level nav items and can only be used with the left sidebar navigation style. It can not be used in conjunction with the [header navigation style](/guides/configuration#navigation-style).
```